tests/js/spec/views/teamCreate.spec.jsx

```diff
@@ -20,13 +20,17 @@ describe('TeamCreate', function() {
     });
   });
 
-  describe('onSubmitSuccess()', function() {
+  describe('handleSubmitSuccess()', function() {
     let wrapper;
-    let locationAssignMock;
+    let redirectMock = jest.fn();
```

Comment:
redirect is a common thing to need to mock out - maybe we could make this a bit easier to do using the router fixture? what you did looks good though

Reply:
Hmm that's true, currently the router fixture uses `sinon.spy`, we could update to use jest mock functions so we can use jest assertions.